clayui.com/content/docs/components/icon.mdx

```diff
@@ -2,10 +2,11 @@
 title: 'Icon'
 description: 'Icons are a visual representation of an idea and/or action.'
 packageNpm: '@clayui/icon'
-sibling: 'docs/components/css-icons.html'
+sibling: 'docs/components/css-icons.mdx'
```

Comment:
I don't think we would want to change this to `.mdx` since we still want to point to the compiled page. Meaning, the page should still be `.html`